tests/unit/source_code/python/test_python_ast.py

```diff
@@ -85,67 +85,41 @@ def test_linter_gets_arg(code, arg_index, arg_name, expected) -> None:
     ],
 )
 def test_args_count(code, expected) -> None:
-    tree = Tree.maybe_parse(code)
-    stmt = tree.first_statement()
+    maybe_tree = MaybeTree.from_source_code(code)
+    stmt = maybe_tree.first_statement()
     assert isinstance(stmt, Expr)
     assert isinstance(stmt.value, Call)
     act = TreeHelper.args_count(stmt.value)
     assert act == expected
 
 
-def test_tree_walks_nodes_once() -> None:
+def test_maybe_tree_walks_nodes_once() -> None:
     nodes = set()
     count = 0
-    tree = Tree.maybe_parse("o.m1().m2().m3()")
-    for node in tree.walk():
+    maybe_tree = MaybeTree.from_source_code("o.m1().m2().m3()")
+    for node in maybe_tree.walk():
         nodes.add(node)
         count += 1
     assert len(nodes) == count
 
 
-def test_parses_incorrectly_indented_code() -> None:
```

Comment:
Test is removed as we now always parse the source code before constructing the tree